python/src/opendp/context.py

```diff
@@ -235,15 +239,25 @@ def loss_of(epsilon=None, delta=None, rho=None, U=None) -> Tuple[Measure, float]
     if epsilon is None and delta is not None:
         raise ValueError("Epsilon must be specified if delta is given.")
 
+    def range_warning(name, value, info_level, warn_level):
+        if info_level < value <= warn_level:
+            logger.info(f'{name} is typically less than {info_level}')
+        if warn_level < value:
+            logger.warning(f'{name} should be less than {warn_level}, and is typically less than {info_level}')
```

Comment:
For delta, the info_level and warn_level are the same which makes the phrasing slightly awkward.
```
delta should be less than 1e-06, and is typically less than 1e-06
``

Add an additional message if the info_level and warn_level are the same